test/WebSites/BasicWebSite/Controllers/HomeController.cs

```diff
@@ -35,11 +35,21 @@ public IActionResult RedirectToActionReturningTaskAction()
             return RedirectToAction("ActionReturningTask");
         }
 
+        public IActionResult RedirectToActionWithFragment()
```

Comment:
This and the method below are not used. Probably should be. I'm not saying we need tests of all 8 new `ControllerBase` overloads but zero is too low. Suggest covering the two methods used here and the longest `RedirectToActionPermanent()` overload.